### PR TITLE
Some examples in Python

### PR DESCRIPTION
**Pull request by anisaAnya**

### CODE REVIEW

File: Python/2-declaration.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#NameError: name 'a' is not defined
+
+a = 5
+print(a)
+type(a)
+#<class 'int'>
+
+b = 5.2
+
+dic = {'d': a,'c': b}
```
Comment (aqrln) [inline Python/2-declaration.py]: nitpick: space after comma

File: Python/3-types.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+a = 5
+print(type(a))
+
+b = 5.2
+type(b)
```
Comment (aqrln) [inline Python/3-types.py]: I suppose it was meant to be `print(type(b))`

**Comment by anisaAnya** [comment]
@aqrln done

**Comment by anisaAnya** [comment]
@tshemsedinov

**Comment by tshemsedinov** [comment]
@DzyubSpirit thanks!

**Pull request merged**